Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -217,19 +217,14 @@ static inline void GeneratePixelShader(T& out, DSTALPHA_MODE dstAlphaMode, API_T
 
 	if (ApiType == API_OPENGL)
 	{
-		// Declare samplers
-		for (int i = 0; i < 8; ++i)
-			out.Write("SAMPLER_BINDING(%d) uniform sampler2DArray samp%d;\n", i, i);
+		out.Write("SAMPLER_BINDING(0) uniform sampler2DArray samp[8];\n");
```

Comment:
I guess this will break https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp#L91